src/modules/RefreshmentMap/content.ts

```diff
@@ -226,3 +241,202 @@ export const ABOUT_ACCORDION_ITEMS = [
     content: dummyParagraph,
   },
 ]
+
+export const SEARCH_SUGGESTIONS: SearchSuggestionItemType[] = [
+  {
+    type: 'Feature',
+    properties: {
+      name: 'Volkspark Prenzlauer Berg',
+      conditions: 'kühl und schattig',
+      category: 'Gruenanlage',
+      bezirk: 'Pankow',
+      bezirksreg: 'Prenzlauer Berg Ost',
+    },
+    geometry: {
+      type: 'Point',
+      coordinates: [13.462956678190112, 52.535800958365236],
+    },
+  },
+  {
+    type: 'Feature',
+    properties: {
+      name: 'Wasserfall am Victoriapark',
```

Comment:
"Viktoriapark" mit a "k"

Reply:
Ok. This comes right from Lisa's GeoJSON. I'll fix that!